packages/boot/src/components/site-hub/index.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import {
+	ExternalLink,
+	Button,
+	// @ts-ignore
+	__experimentalHStack as HStack,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { store as coreStore } from '@wordpress/core-data';
+import { decodeEntities } from '@wordpress/html-entities';
+import { search } from '@wordpress/icons';
+import { rawShortcut } from '@wordpress/keycodes';
+// @ts-expect-error Commands is not typed properly.
+import { store as commandsStore } from '@wordpress/commands';
+import { filterURLForDisplay } from '@wordpress/url';
+import type { UnstableBase } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import SiteIconLink from '../site-icon-link';
+import './style.scss';
+
+function SiteHub() {
+	const { homeUrl, siteTitle } = useSelect( ( select ) => {
+		const { getEntityRecord } = select( coreStore );
+		const _base = getEntityRecord< UnstableBase >(
+			'root',
+			'__unstableBase'
+		);
+		return {
+			homeUrl: _base?.home,
+			siteTitle:
+				! _base?.name && !! _base?.url
+					? filterURLForDisplay( _base?.url )
+					: _base?.name,
+		};
+	}, [] );
+	const { open: openCommandCenter } = useDispatch( commandsStore );
+	const label = __( 'Open command palette' );
+	const shortcutAria = rawShortcut.primary( 'k' );
```

Comment:
Shouldn't this be `displayShortcut`?

Nit: no need to have these vars declared as they are used once.